### PR TITLE
Code Exchange Upgrade

### LINKED ISSUE #50: Code Exchange quality checklist
The purpose of this checklist is to assess the state of this repository, and bring it to an acceptable level for long-term support by completing the tasks listed below.

> Please check every part of this checklist for us to process your request. You can do this by marking them with [x] instead of [ ].

### Documentation

- [x] Twilio [LICENSE](https://github.com/twilio-labs/code-exchange/blob/master/LICENSE) exists in repository

- [x] [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md) exists in repository

- [x] [CONTRIBUTING](https://github.com/twilio-labs/.github/blob/master/CONTRIBUTING.md) exists in repository 

- [x] No broken links in README.md

- [x] Build status exists at top of README.md

- [x] Set up steps are complete and accurate

- [x] CodeExchange repository is linked in README.md

### GitHub

- [x] Dependabot set up

- [x] Dependabot automerge set up

- [x] Uses GitHub Actions (required for Twilio sample) or other CI

### Code

- [x] Packages/dependencies are up to date

- [x] Code compiles against latest packages/dependencies

- [x] Matches [CodeExchange Guidelines](https://github.com/twilio-labs/code-exchange/tree/master/guidelines) to the best of ability

### Tests

- [x] Functionality tests have been created

- [x] All tests pass

### App

- [x] Confirmed sample behaves as expected (after dependencies have been updated)

- [x] This file has been copied into a wiki entry for the respective repository or added as a GitHub issue

### General

- [x] I've read and followed the contribution guidelines

- [x] I've read and implemented my template according to the respective coding guidelines for my project

- [x] I agree and adhere to the Code of Conduct

- [x] The template has been authored by me / or my organization and will be kept up-to-date. If the template will not longer be maintained, I'll notify this project.

### PR DESCRIPTION
**Pull request by po5i**
Resolves #50

### CODE REVIEW

File: .github/workflows/flask.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
```
Comment (bharatmurali) [inline .github/workflows/flask.yml:27]: `Pip install` causes `ERROR: Could not find a version that satisfies the requirement mock==4.0.2`. we should be using `pip3`?

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This demo application shows how to implement instant lead alerts using Python an
 
 [Read the full tutorial here](https://www.twilio.com/docs/tutorials/walkthrough/lead-alerts/python/flask)!
 
-[![Build Status](https://travis-ci.org/TwilioDevEd/lead-alerts-flask.svg?branch=master)](https://travis-ci.org/TwilioDevEd/lead-alerts-flask)
+![](https://github.com/TwilioDevEd/lead-alerts-flask/workflows/Flask/badge.svg)
 
 > We are currently in the process of updating this sample template. If you are encountering any issues with the sample, please open an issue at [github.com/twilio-labs/code-exchange/issues](https://github.com/twilio-labs/code-exchange/issues) and we'll try to help you.
```
Comment (bharatmurali) [inline README.md:14]: please fix numbers of each step below. they all start with `1`

File: README.md [hunk 3]
```diff
@@ -56,8 +56,6 @@ For now, it only runs on Python 2.7 (not 3.4+).
    [Twilio Account Settings](https://www.twilio.com/user/account/settings).
    You will also need a `TWILIO_NUMBER`, which you may find [here](https://www.twilio.com/user/account/phone-numbers/incoming).
 
-   Run `source .env` to export the environment variables.
-
 1. Make sure the tests succeed.
 
     ```bash
```
Comment (bharatmurali) [inline README.md:61]: the `coverage` step wont work without instructions on how to install it. alteratively, we could just run the tests using `python3`

File: README.md [hunk 4]
```diff
@@ -75,5 +73,6 @@ For now, it only runs on Python 2.7 (not 3.4+).
 ## Meta
```
Comment (bharatmurali) [inline README.md:73]: should the above be `python3`?

File: .github/workflows/flask.yml (hunk 1 above)
Comment (po5i) [reply .github/workflows/flask.yml:27]: If you have both Python 2 and 3 installed in your environment, you should use `pip3` instead. In some other environments that uses version management tools such as pyenv then `pip` will be enough.

For the github action workflow, it uses an image with python 3 only so `pip` is good.

I added a note at the beginning of the readme to clarify this confusion.

File: README.md (hunk 3 above)
Comment (po5i) [reply README.md:65]: Coverage works out of the box if the dependencies from requirements.txt are installed. But for the sake of simplifying the instructions, I removed coverage here and use only python.

File: .github/workflows/flask.yml (hunk 1 above)
Comment (bharatmurali) [reply .github/workflows/flask.yml:27]: thank you 👍

File: README.md (hunk 3 above)
Comment (bharatmurali) [reply README.md:65]: oh weird - it didnt work for me for some reason. but cool, this is simpler.

**Pull request merged**